### ISSUE TITLE
Request - Item Stack

### ISSUE THREAD

**Issue by drekbad** [JereKuusela/valheim-item_stand_all_items#9]
Not sure if it would be too hacky or if it has potential to mess with memory usage, but is it possible to add an option to place a stack of an item on the stand? For example, multiple arrows which show up as just one of its type, but the tooltip displays the total count and allows placing and retrieving all at once. Thank you regardless, great mod and it's adding some really amazing opportunities for design/decoration around our base.

**Comment by JereKuusela** [comment]
Yes, that would be possible. It would require everyone to have the mod because people without the mod would take whole stack while only getting one item.
